composer.json

```diff
@@ -25,12 +25,14 @@
         }
     },
     "require": {
-        "php": "^5.6 || ^7.0",
-        "laminas/laminas-zendframework-bridge": "^1.0"
+        "php": "^7.3 || ~8.0.0",
+        "ext-dom": "*",
+        "ext-libxml": "*",
```

Comment:
@froschdesign I've also added these two `ext` dependencies, wdyt?

Reply:
That's good. (Has already been done in other components.) 👍